app/gui/src/ide/integration.rs

```diff
@@ -35,7 +35,13 @@ impl Model {
         // not race for the view.
         *self.project_integration.borrow_mut() = None;
 
-        let project_model = self.controller.current_project();
+        let project_model =
+            if let Some(model) = self.controller.current_project() { model } else { return };
```

Comment:
do not use return. We are not using early returning functions inside of function bodies because it makes the code hard to follow. Use if-then-else instead

Reply:
@wdanilo I think this is not quite accurate, because we use a lot of `?` which are effectively return inside function body.

Still, at this place, +1 for making else block instead of return.

Reply:
`?` and `return` are similar indeed. I would still ban usage of `return` and allow usage of `?`.

Reply:
Fixed